telco-ran/configuration/argocd/example/acmpolicygenerator/acm-group-du-standard-ranGen.yaml

```diff
@@ -41,10 +41,14 @@ policies:
             name: du-ptp-slave
           spec:
             profile:
-                - interface: ens5f0
-                  name: slave
-                  phc2sysOpts: -a -r -n 24
-                  ptp4lOpts: -2 -s --summary_interval -4
+            - interface: ens5f0
+              name: slave
+              phc2sysOpts: -a -r -n 24
+              ptp4lOpts: -2 -s --summary_interval -4
+            recommend:
+            - match:
+              - nodeLabel: node-role.kubernetes.io/master
```

Comment:
On a standard cluster would PTP be tied to the master nodes or the worker nodes? Same below in templated version

Reply:
Ah, that's correct. According to the `mcp` value under the PGT standard [example](https://github.com/openshift-kni/telco-reference/blob/main/telco-ran/configuration/argocd/example/policygentemplates/group-du-standard-ranGen.yaml#L12), PTP should be tied to worker nodes for standard clusters. I've also double checked with the PTP team and they've confirmed.
I've updated the PR accordingly.
Thank you.